static/sass/_pattern_navigation.scss

```diff
@@ -222,5 +337,11 @@
     & ~ .global-nav-dropdown {
       top: 2.1rem !important;
     }
+
+    @media (min-width: $breakpoint-navigation-threshold - 1) {
```

Comment:
I think this breakpoint is being overwritten by the sliding navigation styles. Because the mobile dropdown appears when the desktop view. 
![Screenshot from 2023-07-26 18-35-25](https://github.com/canonical/ubuntu.com/assets/48949356/a211579b-1f30-4ea3-8dc8-26b6176d4d49)

But then it doesn't seem to happen when I navigate to a new page:
![image](https://github.com/canonical/ubuntu.com/assets/48949356/6f8acdbb-2276-40c4-8e05-2dc3846c0f9b)